src/test/java/org/weakref/jmx/AbstractMbeanTest.java

```diff
@@ -153,7 +153,7 @@ public void testOperationInfo(String attribute, boolean isIs, Object[] values, C
             assertEquals(operationInfo.getReturnType(), Object.class.getName(), "Operation Return Type for " + operationName);
             assertEquals(operationInfo.getSignature().length, 1, "Operation Parameter Length for " + operationName);
             MBeanParameterInfo parameterInfo = operationInfo.getSignature()[0];
-            assertEquals(parameterInfo.getName(), "value", "Operation Parameter[0] Name for " + operationName);
+            assertEquals(parameterInfo.getName(), "arg0", "Operation Parameter[0] Name for " + operationName);
```

Comment:
This seems to be an issue created by upgrading to java 17, it doesn't appear to happen if I keep things back at java 8.